frontend/src/pages/RunDetailsV2.tsx

```diff
@@ -294,14 +294,23 @@ function updateToolBarActions(
 }
 
 function getDetailsFields(apiRun?: ApiRun): Array<KeyValue<string>> {
+  // check if the run has finished or not. The default value for apiRun.finished_at is
+  // Date(0), when it is not specified.
+  var finished_at = new Date(0);
```

Comment:
1. I would recommend to use "let" rather than "var".
2. Usually, we use "camelCase" naming convention in TypeScript.

Reply:
+1

Reply:
Thanks for pointing this out! I will change it.